src/util.ts

```diff
@@ -618,7 +618,7 @@ export class Util {
           try {
             authenticatedReqOpts = util.decorateRequest(
               authenticatedReqOpts!,
-              projectId!
+              projectId
```

Comment:
This code is inside of the callback function, `onAuthenticated`, executed after `authClient.getProjectId()`. That is when the `projectId` is assigned using what the authClient says is the correct project ID.